lib/themes/dosomething/paraneue_dosomething/templates/taxonomy/taxonomy-term.tpl.php

```diff
@@ -49,25 +49,26 @@
 
   <section class="container -padded">
     <div class="wrapper">
-      <div class="container__body -narrow">
-        <?php if (!empty($facts)): ?>
+      <?php if (!empty($facts)): ?>
+        <div class="container__body -narrow">
           <h2 class="inline--alt-color"><?php print $facts_header; ?></h2>
           <ul>
             <?php foreach ($facts['facts'] as $fact): ?>
               <li><?php print $fact['fact']; ?> <sup><?php print $fact['footnotes']; ?></sup></li>
             <?php endforeach; ?>
           </ul>
-
-          <section class="sources">
+        </div>
+        <div class="container__body -narrow">
```

Comment:
It can, but then we need to have specific overrides to add that top margin in each scenario. I think it's better to work with the same pattern we use for layout elsewhere. As for semantic concerns, that's why they're placed in the same `<section>`. :)